packages/client/hmi-client/src/components/documents/tera-document-asset.vue

```diff
@@ -267,6 +269,9 @@ const toggleOptionsMenu = (event) => {
 watch(
 	() => props.assetId,
 	async () => {
+		// unsub and sub to avoid multiple subscriptions when switching between document assets
+		await unsubscribe(ClientEventType.Extraction, subscribeToExtraction);
+		await subscribe(ClientEventType.Extraction, subscribeToExtraction);
```

Comment:
Then you should do the `unsubscribe` call within the `sunscribe` function as to not forget to do it in the future.

Reply:
see comment thread below - we will unsubscribe when the component is unmounted